recipeers/medida.py

```diff
@@ -0,0 +1,45 @@
+from enum import Enum
+
+class Medida(Enum):
+    kg = "Kilogramos"
+    g = "Gramos"
+    l = "Litros"
+    ml = "Mililitros"
+    unidades = "Conteo de elementos individuales"
+    cucharadas = "Cucharadas"
+    cucharaditas = "Cucharaditas"
+    pizcas = "Pizcas para cantidades muy pequeñas"
+    pellizcos = "Equivalente a la cantidad que se puede coger con el dedo pulgar e índice"
+    vasos_de_yogurt = "Vasos de yogurt (125 ml aprox)"
+    al_gusto = "Al gusto (saonar)"
+
+def agregar_medidas(conjunto_medidas, nuevas_medidas):
+    """
+    Agrega nuevas medidas al conjunto de medidas existente.
+
+    :param conjunto_medidas: Conjunto (set) de medidas existente.
+    :param nuevas_medidas: Lista de nuevas medidas a agregar.
+    """
+    for nueva_medida in nuevas_medidas:
+        conjunto_medidas.add(nueva_medida)
```

Comment:
¿Vais a trabajar con una variable global para el conjunto de medidas? ¿Una además que declaráis debajo?